addons/hr_recruitment_survey/models/hr_job.py

```diff
@@ -12,7 +12,12 @@ class Job(models.Model):
 
     def action_test_survey(self):
         self.ensure_one()
-        return self.survey_id.action_test_survey()
+        return {
+            'type': 'ir.actions.act_url',
+            'name': "Test Survey",
+            'target': 'new',
+            'url': '/survey/test/%s' % self.survey_id.access_token,
+        }
```

Comment:
hum, better still use survey's function.  In case there are changes in survey's action_test_survey, we don't have to change it manually here as well.

We could do something like 

```
action = self.survey_id.action_test_survey()
action['target'] = 'new'
return action
```

Reply:
good call, i changed it